recipe/meta.yaml

```diff
@@ -23,6 +39,11 @@ requirements:
         - ocl-icd  # [linux]
         - khronos-opencl-icd-loader  # [win]
         - mingwpy  # [win]
+{% if osx_variant == "pocl" %}
+        - osx-pocl-opencl
+        - pocl
```

Comment:
No, not really. Under normal circumstances (i.e. non-Apple), it is the ICD loader's (e.g. `ocl-icd`) responsibility to dispatch to the right implementation. Anything using CL only needs to link against the loader and doesn't need to entangle itself with actual implementation choice. Instead, this choice can safely be deferred to run time. I.e. on Linux, simply install `pocl` and `ocl-icd`, and they'll automatically see and talk to each other. Plus you could install more CL implementations (or add `.icd` files--simple text files with file names--for your built-in GPUs to `$CONDA/etc/OpenCL/vendors`), and they would automatically work.

Apple (AFAICT) doesn't ship a usable ICD loader, and ocl-icd won't compile on Apple, so there's nothing to fill the "runtime implementation multiplexer" role. That's why there's this link-directly-to-pocl crap in here. It's expedient, but it goes against CL's (fairly elegant) multiplexed design.